### DISCUSSION TITLE
eval "$(fzf --bash)" in newer version what do you use for older version?

### DISCUSSION THREAD

**Discussion by alextopic** [junegunn/fzf#4037] [General]
I am using Ubuntu 24.10, and the latest fzf version I can get is version 0.46.1 (debian).
I read that you need version 0.48 and above to use eval "$(fzf --bash)" in you .bashrc file.
What would I put instead of eval "$(fzf --bash)" with my older version?
Any idea when a newer version for Ubuntu/debian systems will be out?

**Comment by LangLangBart** [comment]
There are couple of related issue reports:
- #1866
- #3395
- …

---

> What would I put instead of eval "$(fzf --bash)" with my older version?

This seems to work:

```bash
docker run --workdir=/root --interactive --tty --rm -e FZF_VERSION=0.46.1 ubuntu:24.10 sh -c '
apt-get update && apt-get install -y curl fzf;
curl -o /usr/share/doc/fzf/examples/completion.bash https://raw.githubusercontent.com/junegunn/fzf/refs/tags/${FZF_VERSION}/shell/completion.bash
curl -o /usr/share/doc/fzf/examples/key-bindings.bash https://raw.githubusercontent.com/junegunn/fzf/refs/tags/${FZF_VERSION}/shell/key-bindings.bash
echo ". /usr/share/doc/fzf/examples/completion.bash" >> ~/.bashrc
echo ". /usr/share/doc/fzf/examples/key-bindings.bash" >> ~/.bashrc
/bin/bash'
```

---

> Any idea when a newer version for Ubuntu/debian systems will be out?

no, up to the maintainers of the package.
